.gitignore

```diff
@@ -7,4 +7,5 @@ Makefile
 *.glob
 *.Makefile.d
 .lia.cache
-.DS_Store
\ No newline at end of file
+.DS_Store
+main.v
```

Comment:
Rename main.v to {p4_file_name}_generated.v (and then appropriately update the .gitignore).